icechunk-python/tests/test_can_read_old.py

```diff
@@ -193,7 +193,7 @@ async def test_icechunk_can_read_old_repo():
     ]
     assert sorted(
         [p async for p in store.list_dir("group2/group3/group4/group5/inner")]
-    ) == ["c", "zarr.json"]
+    ) == ["zarr.json"]
```

Comment:
this test never wrote a chunk, so `c` should not have been created.

Reply:
There is some weird behavior in here i think ... sometimes i need it sometimes i don't?